README.md

```diff
@@ -1,3 +1,108 @@
-# Eclipse SmartHome Packaging Sample
+# Eclipse Smarthome Packaging Sample
+<img src="https://www.eclipse.org/concierge/images/logo.png" alt="concierge osgi" width="200"/>
 
-This repo contains a sample of how to create a small working runtime package that uses the Eclipse SmartHome framework.
+This repository contains a sample project which wraps eclipse smarthome within a concierge OSGi container, a lightweight OSGi R5 implementation. You can use this example to build an own minimal distribution with a very optimized memory footprint.
+
+More information about concierge: https://www.eclipse.org/concierge/index.php
+
+1. Prerequisites - Install Maven
+================
+Please use the instructions on main project's readme to install maven: https://github.com/eclipse/smarthome#1-prerequisites
+Make sure **mvn** command is available on your path.
+
+2. Checkout
+================
+Checkout the source code from GitHub, e.g. by running:
+
+```
+git clone https://github.com/eclipse/smarthome-packaging-sample.git
+```
+
+3. Build the distribution
+================
+Run
+```
+mvn clean install
+```
+
+The maven build will create an ZIP file with all required components like
+* the concierge runtime
+* eclipse smarthome bundles
+* 3rd party bundles
+* start scripts
+
+You can find the created distribution under **/target/smarthome-packaging-sample-[version].zip**
+
+### The directory structure of the distribution
+* **addons**: Folder for hotdeployment of bundles
+* **runtime**: Contains the runtime
+ * **concierge**: Contains the conciege osgi runtime
+    * **bundles**: Additional concierge bundles
+    * **framework**: The Framework
+    * **system**: Contains commons and 3rd party bundles
+      * **org.eclipse.jetty**: All jetty bundles
+      * **org.eclipse.smarthome**: All smarthome bundles
+ * **etc**: Quartz configuration, Jetty configuration, keystore
+* **userdata**: This folder is created during the first startup and contains persistent userdata and the osgi storage.
+
+4. Start runtime
+================
+Extract the distribution zip file and start the runtime:
+```
+unzip smarthome-packaging-sample-[version].zip
+cd runtime/concierge
```

Comment:
would it be possible to move the `start.sh` script to the root, so that this line is not needed? Or create a new `start.sh`in the root that does nothing else than cd'ing and launching the other `start.sh`.